### ISSUE TITLE
@readonly secondary tag would be useful for @property primary tag

### ISSUE THREAD

**Issue by zpapp** [yui/yuidoc#123]
Currently the @readonly secondary tag is ignored if the primary tag is @property. I would like to be able to add the "readonly" flag to properties as well, not just to @attribute and @config elements.

**Comment by lannymcnie** [comment]
I agree that this is an issue. In JavaScript, we typically treat properties as read-only. Marking them as final doesn't reflect the same meaning. I would like this support as well.

(thanks for an awesome tool)

**Comment by okuryu** [comment]
OK, I'll take a look.

**Comment by lannymcnie** [comment]
We modified the source locally to make it work - I can submit a pull request tomorrow.

**Comment by okuryu** [comment]
Sure, patch welcome. :)

**Comment by andyearnshaw** [comment]
@lannymcnie are you still able to submit a pull request for this?

**Comment by okuryu** [comment]
Maybe not yet.
